compiler/rustc_error_codes/src/error_codes/E0546.md

```diff
@@ -13,15 +13,15 @@ fn unstable_fn() {}
 fn stable_fn() {}
 ```
 
-To fix the issue you need to provide a feature name.
+To fix the issue you need to provide the `feature` field.
 
 ```
 #![feature(staged_api)]
 #![stable(since = "1.0.0", feature = "test")]
 
-#[unstable(feature = "unstable_fn", issue = "none")] // ok!
+#[unstable(feature = "unstable_fn", issue = "none")]
```

Comment:
Please keep the `// ok!`.

Reply:
Ok, I'm going to add it also here then. Isn't it? :)

https://github.com/rust-lang/rust/pull/81830

Reply:
Arf indeed! Sorry, didn't pay enough attention...

Reply:
Don't worry, I think now I have everything to do it properly in the net PRs. Thanks!